runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -516,7 +520,16 @@ public void createGameTimer(GameTimer timer)
 		removeGameTimer(timer);
 
 		TimerTimer t = new TimerTimer(timer, this);
-		t.setTooltip(timer.getDescription());
+
+		if (timer.getDescription().equals(("Cannon")))
```

Comment:
This is probably wrong way to do this, extend the existing API instead

Reply:
Any way you would suggest changing this? I'm not really sure what to do differently